### DISCUSSION TITLE
Seams location questions

### DISCUSSION THREAD

**Discussion by DzzD** [supermerill/SuperSlicer#1816] [Q&A] [answered]
Hi,

There are four options to select seams position under "Perimeters & Shell=>Seam", Unfortunatly there is no option available to set a position x,y (as in S3D), is it planned ?

I have trouble to understand some options of this parameter, for example "Aligned" mean what ? aligned where ?

What should I do if I want a clean vertical seam on a cilinder for example ?

**Comment by supermerill** [comment, marked as answer]
You can paint the seam
![image](https://user-images.githubusercontent.com/6536403/140642406-389cd074-0d99-468d-a495-e57d1298b82c.png)
you can also define a point to attract them via a seam modifier (right-clic). Maybe a bit more precise.
![image](https://user-images.githubusercontent.com/6536403/140642480-5b18ab5f-9a3c-4f0b-ba18-8f89c50a4522.png)

**Comment by DzzD** [comment]
Ha ok ! very nice, I have seen the "add seam position", but did not understand well what was this big sphere.

About painting seams, it is pretty hard to draw a straight vertical line

Thanks for all thoses informations

**Comment by DzzD** [comment]
Just one more question, why did you choose a sphere for the seam attractor rather than a vertical pillar ? regarding why it is for, the  sphere is very disturbing for me.

**Reply by supermerill** [reply]
Because the algorithm only sees a point (the geometric center). on an object, on a layer, it will use the x,y of the seam sphere that has the nearest z. If you have only one seam sphere on the object, the z position isn't relevant.

Ideally, you should be able to add your own seam object, and I should parse it to get the geometric mean for each layer, but it's not made. Too much work for the added benefit. Maybe one day...

**Reply by DzzD** [reply]
So basically it use the sphere x,y pos (center indeed) for each layer ? 

EDIT: IMHO, it is not an object that is necessary but rather a path that goes along z axis (something editable for each z like variable layer height tool)

**Reply by supermerill** [reply]
> So basically it use the sphere x,y pos (center indeed) for each layer ?

If you have only one, yes.
The seam will be at the nearest possible position from this x,y.
